agent/04_agent_configure.sh

```diff
@@ -17,9 +17,16 @@ early_deploy_validation
 
 export CLUSTER_NAMESPACE=${CLUSTER_NAMESPACE:-"cluster0"}
 
+function add_host_entry {
+    ip=${1}
+    hostname=${2}
+
+    echo "${hostname}">>"${OCP_DIR}"/hostname
```

Comment:
I think the OCP_DIR has already been removed this point https://github.com/openshift-metal3/dev-scripts/blob/master/ocp_cleanup.sh#L19

Instead of using a file, should be able to get the list of hostnames that were added to dns using, for example:
`$ sudo virsh net-dumpxml ostestbm | xmllint --xpath "//dns/host/hostname" -`